src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -2822,6 +2873,14 @@ void InitSolarHeatGains(EnergyPlusData &state)
 
         for (int zoneNum = 1; zoneNum <= state.dataGlobal->NumOfZones; ++zoneNum) {
             for (int SurfNum : state.dataHeatBal->Zone(zoneNum).ZoneExtSolarSurfaceList) {
+
+                Real64 GndSurfReflectance = 0.0;
+                if (state.dataSurface->UseSurfPropertyGndSurfRefl(SurfNum)) {
+                    GndSurfReflectance = state.dataSurface->GroundSurfsProperty(state.dataSurface->GroundSurfsPropertyNum(SurfNum)).SurfsReflAvg;
+                } else {
+                    GndSurfReflectance = state.dataEnvrn->GndReflectance;
+                }
+
```

Comment:
Ground reflectance value from the new Ground surfaces object `SurfaceProperty:GroundSurfaces` is used if specified, instead of the default global.